backend/Repositories/WhaleRepo.cs

```diff
@@ -12,5 +18,11 @@ public WhaleRepo(WhaleSpottingDbContext context)
         {
             _context = context;
         }
+         public IEnumerable<Species> GetAllSpecies()
+        {
+            return _context.Species
+                .Include(s => s.ConservationStatus)
+                .OrderByDescending(s => s.Name);
```

Comment:
Why descending? Reverse alphabetical order seems like a strange choice, wouldn't ascending make more sense?

Reply:
Agreed! We couldn't work out how to do it!

Reply:
Please help!!!!